custom/icds_reports/ucr/reports/ls/ls_v2_thr_30_days.json

```diff
@@ -0,0 +1,131 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-test",
+    "icds-sql",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "report_id": "ls_v2_thr_30_days",
+  "data_source_table": "static-ccs_record_cases",
+  "config": {
+    "title": "UCR V2 - LS - THR in Last 30 days",
```

Comment:
This name seems misleading. This is only CCS cases (not child health which also get thr), and I dont see where the 30 day filtering is.

Reply:
In looking at the data source, it looks like the 30 days are "30 days since the case was last updated". Is that intended or is it supposed to be 30 days since today (or current month like many other reports)? In practice, that means if a ccs record case has not been updated since January, but it got more than 21 days of THR on that date, it will appear here as having received greater than 21 THR in the past 30 days.